demos/code/index.html

```diff
@@ -11,8 +11,8 @@
   <script src="../../javascript_compressed.js"></script>
   <script src="../../python_compressed.js"></script>
   <script src="../../php_compressed.js"></script>
-  <script src="../../lua_compressed.js"></script>
```

Comment:
The changes in this file should be reverted.  But this file does need the sort block added to its toolbox XML.